_api-reference/index-apis/clone.md

```diff
@@ -55,6 +55,8 @@ Parameter | Type | Description
 wait_for_active_shards | String | The number of active shards that must be available before OpenSearch processes the request. Default is 1 (only the primary shard). Set to all or a positive integer. Values greater than 1 require replicas. For example, if you specify a value of 3, the index must have two replicas distributed across two additional nodes for the operation to succeed.
 master_timeout | Time | How long to wait for a connection to the master node. Default is `30s`.
 timeout | Time | How long to wait for the request to return. Default is `30s`.
+wait_for_completion | Boolean | When set to `false`, the request will return immediately instead of waiting for the operation to finish. To monitor the operation status, use the [Tasks API]({{site.url}}{{site.baseurl}}/api-reference/tasks/) with the task ID that the request returns. Default is `true`.
```

Comment:
```suggestion
wait_for_completion | Boolean | When set to `false`, the request will be returned immediately instead of after the operation is finished. To monitor the operation status, use the [Tasks API]({{site.url}}{{site.baseurl}}/api-reference/tasks/) with the task ID returned by the request. Default is `true`.
```